### PR TITLE
Footer and backdrop styling

### PR DESCRIPTION
**Pull request by jgiwinski**
### Is this a fix or a feature?
* Fix
### What is the change?
* Add backdrop poster to movie details view
* Clean up and delete un used code and files
* Adjust responsiveness for mobile view 
* Style footer 
* Change search bar background image 
* Small styling changes to site title, budget/revenue, and release date

### Where should the reviewer start?
* MovieDetails.js and movieDetails.css
### Relevant tickets
* https://github.com/percworld/tomatillos/projects/1#card-58324361

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -86,6 +85,7 @@ class App extends Component {
                   showFeatured={this.showFeatured}
                   handleSearchEntry={this.handleSearchEntry}
                   searchByWord={this.searchByWord}
+                  disabled={!this.state.value}
```
Comment (percworld) [inline src/Components/App/App.js:88]: We won't need this disabled prop

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import './Footer.css';
+import { FaGithub } from 'react-icons/fa';
```
Comment (percworld) [inline src/Components/Footer/Footer.js:3]: Great idea!

**Review by percworld — CHANGES_REQUESTED** [review]
Great work, one thing to fix

**Comment by percworld** [comment]
Component : App.js
Rm  disabled={!this.state.value}

File: src/Components/App/App.js (hunk 1 above)
Comment (jgiwinski) [reply src/Components/App/App.js:88]: Lets remove it on the next PR - my branch is all weird because of the merge conflicts

**Review by percworld — APPROVED** [review]

**Pull request merged**